Source/Diagnostics/Diagnostics.cpp

```diff
@@ -209,6 +213,18 @@ Diagnostics::InitData ()
         m_all_species.clear();
         amrex::Print() << " WARNING: For full diagnostics on a reduced domain, particle io is not supported, yet! Therefore, particle-io is disabled for this diag " << m_diag_name << "\n";
     }
+
+    // default for writing species output is 1
+    int write_species = 1;
+    pp.query("write_species", write_species);
+    if (write_species == 0) {
+        if (m_format == "checkpoint"){
+            amrex::Abort("For checkpoint format, write_species flag must be 1.");
+    }
+    // if user-defined value for write_species == 0, then clear species vector
+        m_species_names.clear();
+    m_all_species.clear();
```

Comment:
```suggestion
    m_species_names.clear();
    m_all_species.clear();
```

Reply:
thank you! I missed seeing this incorrect indent!